src/Request/request.js

```diff
@@ -0,0 +1,17 @@
+import { bookStore } from '../Utils/request';
+
+const requestGetBook = () => {
+    return bookStore.get(`books`);
```

Comment:
Why did you use this `books` and not 'books', are you going to concatenate something?